api/WheelEvent.json

```diff
@@ -305,45 +257,45 @@
           }
         }
       },
-      "DOM_DELTA_PAGE": {
+      "deltaY": {
         "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/WheelEvent/DOM_DELTA_PAGE",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/WheelEvent/deltaY",
           "support": {
             "webview_android": {
-              "version_added": null
+              "version_added": true
             },
             "chrome": {
-              "version_added": null
+              "version_added": "31"
             },
             "chrome_android": {
-              "version_added": null
+              "version_added": true
             },
             "edge": {
-              "version_added": null
+              "version_added": true
             },
             "edge_mobile": {
-              "version_added": null
+              "version_added": true
             },
             "firefox": {
-              "version_added": null
+              "version_added": "17"
             },
             "firefox_android": {
-              "version_added": null
+              "version_added": "17"
             },
             "ie": {
-              "version_added": null
+              "version_added": "9"
```

Comment:
Assume we're not adding a notes field here for ie?

Reply:
ditto